Production/govuk_ios/Views/Search/SearchErrorView.swift

```diff
@@ -60,8 +60,7 @@ class SearchErrorView: UIView {
         errorDescription.text = errorDesc
         errorLink = link
 
-        guard link != nil
-        else { return errorLinkButton.isHidden = true }
+        link == nil ? (errorLinkButton.isHidden = true) : (errorLinkButton.isHidden = false)
```

Comment:
Could just do this also I think? `errorLinkButton.isHidden = link == nil`

Reply:
Updated